tests/functional/test_install_force_reinstall.py

```diff
@@ -22,8 +23,12 @@ def check_force_reinstall(script, specifier, expected):
     result = script.pip_install_local('simplewheel==1.0')
     check_installed_version(script, 'simplewheel', '1.0')
 
+    # Remove an installed file to test whether --force-reinstall fixes it.
+    script.site_packages_path.joinpath("simplewheel", "__init__.py").unlink()
```

Comment:
I would assign this to a separate variable and delete it using that, like

```python
to_fix = script.site_packages_path.joinpath("simplewheel", "__init__.py")
to_fix.unlink()

# ...

assert str(to_fix) in ...
```

Reply:
The problem? here is actually that `site_packages_path` is absolute and `site_packages` is relative, and I couldn’t find a clean way to use `in` from the latter (the proposed check fails on my computer), or call `os.unlink()` on the former.

Reply:
One way I found is

```python
assert os.path.relpath(to_fix, script.base_path) in ...
```

I will make the change if this looks better than the current implementation.

Reply:
I think it does look better, since the relationships are much more obvious. I think knowing offhand that `_path` is absolute and non-`_path` is relative would be a lot to ask of a reader, but the new construction is better than a comment.